styles.css

```diff
@@ -0,0 +1,176 @@
+*{
+	box-sizing:border-box;
+}
+
+/*Header and links*/
+
+.container {
+	position:relative;
+	width:90%;
+	border:5px solid #000000;
+	overflow:auto;
+	margin:0 auto;
+}
+
+.container ul{
+	display:inline-block;
+	float:left;
+	width:25%;
+	padding:0;
+}
+
+.navLinks li,
+.social a{
+	display:inline-block;
+	margin:0 auto;
+	width:33.3%;
+	float:left;
+	text-align:center;
+	color:#000000;
+	border:2px solid #ffffff;
+}
+
+.navLinks li a,
+.social a {
+	text-decoration:none;
+	color:#000000;
+	padding:3px;
+}
+
+
+.title{
+	border:5px solid #000000;
+	width:50%;
+	display:inline-block;
+	float:left;
+	margin:10px 0 0 0;
+	text-align:center;
+	z-index:5;
+	background-color:#ffffff;
+}
+
+.social{
+	display:inline-block;
+	float:right;
+	width:25%;
+}
+
+.navLinks,
+.social{
+	height:50px;
+	margin-top:50px;
+}
+
+.navlinks,
+.headImage{
+	background-image:url("headImage.jpg");
+	height:400px;
+	margin-top:80px;
+	z-index:3;
+	background-size:cover;
+	background-repeat:no-repeat;
+}
+
+.headImage h3{
+	position:absolute;
+	color:#ffffff;
+	bottom:0;
+	text-align:center;
+	border:2px solid #ffffff;
+	margin:10px;
+}
+
+.navLinks li a:hover{
+	border:2px solid #000000;
+}
+
+a[href="index.html"]{
+	border:2px solid #000000;
+}
+
+.social a{
+	height:30px;
+	background-repeat:no-repeat;
+	background-position:center;
+	margin-bottom:1px;
+	background-height:100%;
+}
+
+.twitter{
+	background-image:url("twitter.svg");
+}
+
+.github{
+	background-image:url("github.svg");
+}
+
+.linkedIn{
+	background-image:url("linkedIn.svg");
+}
+
+.title h4{
+	color:#808080;
+}
+
+
+/*content*/
+
+main.container{
+	height:1300px;
```

Comment:
can `height` be a `vh`?
also maybe try `min-height` attribute so you can set a height that can go bigger if content is too big for container.

Reply:
changed to min-height